src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid $main-border-color;
```

Comment:
Make sure to follow the DRY (Don't Repeat Yourself) principle. Since you are using `$main-border-color` for the border color, consider creating a variable for border width and border style if they are reused throughout your styles.